templates/systems/element_detail_control.html

```diff
@@ -81,7 +81,7 @@ <h4 class="usa-accordion__heading">
             {% if c.implementation %}
               <p>{{ c.implementation|linebreaks }}</p>
             {% else %}
-              <p>No Implementation Standards provide.</p>
+              <p>No Implementation Standards provided.</p>
```

Comment:
Good catch

Reply:
Sadly, the Responsibilities live in the database and then need to be assigned to Components before they are added to a project. It is one of the things that we are going to need to address going forward.

Reply:
Ah, now I understand. I moved the template out of /branding, but in your local it is still there so it is overriding the version in /templates. We need to abandon the /branding directory and move everything into /templates. I'll create a ticket.